recipes/sdl/all/conanfile.py

```diff
@@ -172,20 +177,21 @@ def build_requirements(self):
             self.build_requires("wayland/1.20.0")  # Provides wayland-scanner
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+        get(self, **self.conan_data["sources"][self.version], strip_root=True,
+            destination=self._source_subfolder)
 
     def _patch_sources(self):
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            tools.patch(**patch)
+        for data in self.conan_data.get("patches", {}).get(self.version, []):
+            patch(self, **data)
```

Comment:
```suggestion
        apply_conandata_patches(self)
```

Reply:
oh wow, wasn't aware of such function. Would be great to have to documented at https://github.com/conan-io/conan-center-index/blob/master/docs/v2_linter.md